### PR TITLE
GEN-1054: Goerli v1.3.1 deployment

### PR DESCRIPTION
**Pull request by fedealconada**
## Motivation

Deploy latest contracts to Goerli

## Solution

Deploy v1.3.1 on Goerli

## Implementer Checklist

- [ ]  [FIRST TIME ONLY] Review the [Solcurity](https://github.com/Rari-Capital/solcurity) standard
- [ ]  [OPTIONAL] Create a reference implementation (in python or JS)
- [x]  Create a short "Motivation" section for the PR
- [ ]  Write a spec for the feature and put it in the PR description – basic function names and expected state transitions is OK
- [ ]  Check all of the new revert paths with concrete tests
- [ ]  Check all of the new storage slot writes with concrete tests – do not make the passing test case the trivial case!
- [ ]  Go line-by-line through the new code and ensure it has all been covered by concrete tests – since we don’t have a coverage engine for foundry yet, you can imagine how one might check that each different code path is covered
- [ ]  Simplify the implementation and spend some time trying to minimize gas costs
- [ ]  Write fuzz tests for the feature – try everything to break the implementation (is this function monotonically in/decreasing, should it always be less than something else, etc)
- [ ]  Capture bugs discovered during the above steps in concrete tests (regression tests)
- [ ]  Add integration tests – fuzz or concrete tests that test how the new code affects the entire system, either locally or with a mainnet fork
- [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
- [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
- [ ]  Get the PR reviewed by at least two people

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,14 +53,14 @@ sY-cUSDC:01-07-2022:6 | [0x4ACA82E5686226A679875AACde7ECdf5fC5477ec](https://eth
 
 \* We are aware that the Name and Symbol does not include the Space LP Share. This will be fixed in the next version of Space.
 
-### Goerli v1.2.0
+### Goerli v1.3.0
```
Comment (jparklev) [inline README.md]: Should this be 1.3.1?

File: pkg/deployments/deploy/simulated/05-emergency-stop.js [hunk 2]
```diff
@@ -18,15 +18,15 @@ module.exports = async function () {
     log: true,
   });
 
+  const emergency = await ethers.getContract("EmergencyStop", signer);
   log("Trust the emergency address on the divider");
   if (!(await divider.isTrusted(emergencyAddress))) {
```
Comment (jparklev) [inline pkg/deployments/deploy/simulated/05-emergency-stop.js:23]: Could also do `emergency.address` with this change

**Review by jparklev — APPROVED** [review]
👊

File: pkg/deployments/deploy/simulated/05-emergency-stop.js (hunk 2 above)
Comment (fedealconada) [reply pkg/deployments/deploy/simulated/05-emergency-stop.js:23]: Yeah, we need that line because its then usesd in line 28

File: README.md (hunk 1 above)
Comment (abylaw) [inline README.md]:
```suggestion
### Goerli v1.3.1
```
?

**Review by abylaw — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (abylaw) [reply README.md]: lol oops already mentioned 😄

**Pull request merged**